modules/fedimint-mint-client/src/lib.rs

```diff
@@ -154,6 +155,44 @@ impl OOBNotes {
             .expect("Invariant violated: OOBNotes does not contain any notes")
     }
 
+    pub fn notes_json(&self) -> Result<serde_json::Value, serde_json::Error> {
+        let mut notes_map = serde_json::Map::new();
+        for notes in self.0.iter() {
+            match notes {
+                OOBNotesData::Notes(notes) => {
+                    let notes_json = serde_json::to_value(notes)?;
+                    notes_map.insert("notes".to_string(), notes_json);
```

Comment:
We should probably assert that insert returns `None` to avoid confusing ourselves should we ever allow multiple instances of some data type.